nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/bigquery/AbstractBigQueryProcessor.java

```diff
@@ -82,23 +80,14 @@
             .addValidator(StandardValidators.NON_EMPTY_EL_VALIDATOR)
             .build();
 
-    public static final PropertyDescriptor TABLE_SCHEMA = new PropertyDescriptor.Builder()
-            .name(BigQueryAttributes.TABLE_SCHEMA_ATTR)
-            .displayName("Table Schema")
-            .description(BigQueryAttributes.TABLE_SCHEMA_DESC)
-            .required(false)
-            .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
-            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
-            .build();
-
-    public static final PropertyDescriptor READ_TIMEOUT = new PropertyDescriptor.Builder()
-            .name(BigQueryAttributes.JOB_READ_TIMEOUT_ATTR)
-            .displayName("Read Timeout")
-            .description(BigQueryAttributes.JOB_READ_TIMEOUT_DESC)
+    public static final PropertyDescriptor IGNORE_UNKNOWN = new PropertyDescriptor.Builder()
+            .name(BigQueryAttributes.IGNORE_UNKNOWN_ATTR)
+            .displayName("Ignore Unknown Values")
+            .description(BigQueryAttributes.IGNORE_UNKNOWN_DESC)
             .required(true)
-            .defaultValue("5 minutes")
-            .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
-            .addValidator(StandardValidators.TIME_PERIOD_VALIDATOR)
+            .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
+            .allowableValues("true", "false")
```

Comment:
Do you actually need it? 
Actually you further limit the values, boolean validator ignores case.